pandas/tests/plotting/test_frame.py

```diff
@@ -231,14 +231,42 @@ def test_plot_xy(self):
     @pytest.mark.slow
     def test_logscales(self):
         df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
         ax = df.plot(logy=True)
         self._check_ax_scales(ax, yaxis='log')
+        assert ax.get_yscale() == 'log'
+
+        ax = df.plot(logy='sym')
+        self._check_ax_scales(ax, yaxis='symlog')
+        assert ax.get_yscale() == 'symlog'
 
         ax = df.plot(logx=True)
         self._check_ax_scales(ax, xaxis='log')
+        assert ax.get_xscale() == 'log'
+
+        ax = df.plot(logx='sym')
+        self._check_ax_scales(ax, xaxis='symlog')
+        assert ax.get_xscale() == 'symlog'
 
         ax = df.plot(loglog=True)
         self._check_ax_scales(ax, xaxis='log', yaxis='log')
+        assert ax.get_xscale() == 'log'
+        assert ax.get_yscale() == 'log'
+
+        ax = df.plot(loglog='sym')
+        self._check_ax_scales(ax, xaxis='symlog', yaxis='symlog')
+        assert ax.get_xscale() == 'symlog'
+        assert ax.get_yscale() == 'symlog'
+
+    @pytest.mark.parametrize("wrong_input", ["sm", "symlog"])
+    @pytest.mark.parametrize("input_param", ["logx", "logy", "loglog"])
+    def test_invalid_logscale(self, wrong_input, input_param):
+        # GH: 24867
+        df = DataFrame({'a': np.arange(100)}, index=np.arange(100))
+
+        msg = "Valid inputs are boolean, None and 'sym'"
```

Comment:
Can you update the match msg to ensure that the bad parameter is included. You can remove the "wrong_input" fixture I think, and just test one bad input.

Reply:
updated!